api_app/analyzers_manager/file_analyzers/cape_sandbox.py

```diff
@@ -204,17 +207,51 @@ def __poll_for_result(
         self,
         task_id,
     ) -> dict:
+        logger.info(
+            f" Job: {self.job_id} -> "
+            f"The analysis starts in 30 seconds on average, sleeping..."
+        )
+        time.sleep(30)  # empirical value
+
+        # decreasing timeout:
+        #   real analysis duration with poll distance of 1 minute + the remaining module
+        #   empirical value of processing duration time
+        #   real polling with max tries
+        #   final attempts in case there is a bottleneck
+        timeout_attempts = (
+            [60] * (self.timeout // 60)
+            + [self.timeout % 60]
+            + [50]
+            + [self.poll_distance] * self.max_tries
+            + [60] * 3
+        )
```

Comment:
What the fuck is this ahahahaha

Reply:
I'd like to have a point interpolation function but those are empirical magic numbers as to when you are most likely to get a result from the analysis.

Reply:
I could handle it by doing many `if` inside the `for` but I preferred to enclose everything in this data structure.
I would greatly appreciate suggestions because I know the code isn't easily readable like that

Reply:
I was thinking if there was a way to tell Intelowl that the analysis is finished instead of doing polling, or maybe a mix of the two